workflow-service/src/main/java/com/redhat/parodos/project/dto/response/ProjectAccessRequestDTO.java

```diff
@@ -0,0 +1,29 @@
+package com.redhat.parodos.project.dto.response;
+
+import java.util.UUID;
+
+import com.redhat.parodos.project.enums.ProjectAccessStatus;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ProjectAccessRequestDTO {
```

Comment:
It might be useful to add the project access request create date. WDYT

Reply:
I think this is fine. ProjectAccessRequest entity doesn't have it. also UI doesn't show it neither